sys/include/net/l2filter.h

```diff
@@ -62,7 +62,7 @@ extern "C" {
  * addresses at the same time.
  */
 typedef struct {
-    uint8_t addr[L2FILTER_ADDR_MAXLEN];     /**< link layer address */
+    uint8_t addr[CONFIG_L2FILTER_ADDR_MAXLEN];     /**< link layer address */
     size_t addr_len;                        /**< address length in byte */
```

Comment:
Please keep the documentation aligned here :)

Reply:
:+1: